src/zip.c

```diff
@@ -39,7 +39,7 @@
 #include "miniz.h"
 #include "zip.h"
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || defined(_WIN32) || defined(_WIN64)
```

Comment:
can we add `ifdef` for your compiler, because this change opens the _if branch_ for all compilers on windows, and I'm not sure it's a good approach.

Reply:
ok, added